--- library/Director/IcingaConfig/IcingaConfigHelper.php	2022-01-21 11:17:18.767360715 +0200
+++ library/Director/IcingaConfig/IcingaConfigHelper.php	2022-01-21 11:16:21.061597179 +0200
@@ -123,31 +123,39 @@
     //       Parameter? Dedicated method? Always if \n is found?
     public static function renderString($string)
     {
-        $special = [
-            '/\\\/',
-            '/"/',
-            '/\$/',
-            '/\t/',
-            '/\r/',
-            '/\n/',
-            // '/\b/', -> doesn't work
-            '/\f/',
-        ];
-
-        $replace = [
-            '\\\\\\',
-            '\\"',
-            '\\$',
-            '\\t',
-            '\\r',
-            '\\n',
-            // '\\b',
-            '\\f',
-        ];
+        if(substr($string, 0, 2) === "@@"){
 
-        $string = preg_replace($special, $replace, $string);
+            return substr($string, 2);
 
-        return '"' . $string . '"';
+        } else {
+
+            $special = [
+                '/\\\/',
+                '/"/',
+                '/\$/',
+                '/\t/',
+                '/\r/',
+                '/\n/',
+                // '/\b/', -> doesn't work
+                '/\f/',
+            ];
+
+            $replace = [
+                '\\\\\\',
+                '\\"',
+                '\\$',
+                '\\t',
+                '\\r',
+                '\\n',
+                // '\\b',
+                '\\f',
+            ];
+
+            $string = preg_replace($special, $replace, $string);
+
+            return '"' . $string . '"';
+
+        }
     }
 
     public static function renderPhpValue($value)
